src/Frontend/Document.php

```diff
@@ -131,14 +132,20 @@ class Document implements Renderable
      */
     protected $forumApiDocument;
 
+    /**
+     * @var Request
+     */
+    protected $request;
+
     /**
      * @param Factory $view
      * @param array $forumApiDocument
      */
-    public function __construct(Factory $view, array $forumApiDocument)
+    public function __construct(Factory $view, array $forumApiDocument, Request $request)
```

Comment:
Could this be considered a breaking change?

Reply:
Never mind, looks like it's only initialized in one place.

Reply:
Well it would be a breaking change if an extension decided to extend it... Though I don't see why an extension would try to extend that class and I don't think anyone does it :sweat_smile:

Reply:
Yeah, I didn't look at other uses of Frontend\Document until after commenting, which is why I resolved the conversation.